_includes/manuals/1.23/1.23.0-RC1_release_notes.md

```diff
@@ -0,0 +1,303 @@
+
+#### Compute resources - GCE
+* Fix API endpoints for  GCE Compute Resource ([#27338](https://projects.theforeman.org/issues/27338))
+* On host edit form, it doesn't show selected image value in GCE ([#27316](https://projects.theforeman.org/issues/27316))
+* [UI] External IP value is not being saved for GCE Compute Profile ([#27157](https://projects.theforeman.org/issues/27157))
+* The VM power off dialog shows unusual name of VM. ([#26814](https://projects.theforeman.org/issues/26814))
+* The VM power off/on from CR -&gt; VMs tab, throws undefined method error 'state' ([#26812](https://projects.theforeman.org/issues/26812))
+* Show Console output for a instance created using GCE ([#26787](https://projects.theforeman.org/issues/26787))
+* Associate vms not working for google compute resource ([#26761](https://projects.theforeman.org/issues/26761))
+* Invalid key on VM configuration edit from google console for VM provisioned through Foreman ([#26599](https://projects.theforeman.org/issues/26599))
+* The deleted additinal disk from Foreman is being created in VM in GCE ([#26442](https://projects.theforeman.org/issues/26442))
+
+#### Compute resources - OpenStack
+* unintialized Constant in app/models/compute_resources/foreman/model/openstack.rb ([#26971](https://projects.theforeman.org/issues/26971))
+* OpenStack 401 when project domain ID is missing ([#26664](https://projects.theforeman.org/issues/26664))
+
+#### Compute resources - VMware
+* VMware resource pools are populated only from first cluster found ([#27363](https://projects.theforeman.org/issues/27363))
+* Annotation notes are not set for VMware mages based builds ([#27313](https://projects.theforeman.org/issues/27313))
+* Cloud init template missing redhat_register snippet ([#27276](https://projects.theforeman.org/issues/27276))
+* Deploying a New Host to vmware compute resource from existing template always ends up with thin provisioned disk ([#27015](https://projects.theforeman.org/issues/27015))
+* Update VMware Hardware version compatible with 6.7 and above ([#26849](https://projects.theforeman.org/issues/26849))
+* Update new guest OS type supported on VMware vSphere 6.7 and above ([#26838](https://projects.theforeman.org/issues/26838))
+* VMware compute resource cannot find VM folders when datacenters are themselves nested in folders ([#25849](https://projects.theforeman.org/issues/25849))
+* Associating VMs/hosts with multiple VMWare compute resources from same vcenter server ([#15626](https://projects.theforeman.org/issues/15626))
+
+#### Compute resources - oVirt
+* Remove hardcoded max values of oVirt CR ([#27334](https://projects.theforeman.org/issues/27334))
+* Remove "experimental" from ovirt v4 checkbox ([#26840](https://projects.theforeman.org/issues/26840))
+* Allow Satellite user to choose between VNC, SPICE for Display type on RHEV compute resources ([#24532](https://projects.theforeman.org/issues/24532))
+
+#### Core
```

Comment:
These should be moved to `#### Smart proxies` section, as they are the core component of smart proxy

Reply:
done!